lib/src/commands/create/create.dart

```diff
@@ -1,314 +1,57 @@
-import 'dart:async';
-
-import 'package:args/args.dart';
 import 'package:args/command_runner.dart';
 import 'package:mason/mason.dart';
-import 'package:meta/meta.dart';
-import 'package:path/path.dart' as path;
-import 'package:universal_io/io.dart';
 import 'package:usage/usage_io.dart';
-import 'package:very_good_cli/src/command_runner.dart';
-import 'package:very_good_cli/src/commands/create/templates/templates.dart';
-
-const _defaultOrgName = 'com.example.verygoodcore';
-const _defaultDescription = 'A Very Good Project created by Very Good CLI.';
-
-final _templates = [
-  VeryGoodCoreTemplate(),
-  DartPkgTemplate(),
-  FlutterPkgTemplate(),
-  FlutterPluginTemplate(),
-  VeryGoodDartCLITemplate(),
-  VeryGoodDocsSiteTemplate(),
-  VeryGoodFlameGameTemplate(),
-];
-
-final _defaultTemplate = _templates.first;
-
-// A valid Dart identifier that can be used for a package, i.e. no
-// capital letters.
-// https://dart.dev/guides/language/language-tour#important-concepts
-final RegExp _identifierRegExp = RegExp('[a-z_][a-z0-9_]*');
-final RegExp _orgNameRegExp = RegExp(r'^[a-zA-Z][\w-]*(\.[a-zA-Z][\w-]*)+$');
-
-/// A method which returns a [Future<MasonGenerator>] given a [MasonBundle].
-typedef MasonGeneratorFromBundle = Future<MasonGenerator> Function(MasonBundle);
-
-/// A method which returns a [Future<MasonGenerator>] given a [Brick].
-typedef MasonGeneratorFromBrick = Future<MasonGenerator> Function(Brick);
+import 'package:very_good_cli/src/commands/create/commands/commands.dart';
+import 'package:very_good_cli/src/commands/create/create_subcommand.dart';
```

Comment:
do we need this import or can it be added to the commands.dart barrel?

Reply:
Adding to the barrel